app/controllers/search_controller.rb

```diff
@@ -3,6 +3,15 @@ class SearchController < ApplicationController
   before_action :format_integer_params
   before_action :sanitize_params, only: %i[listings reactions feed_content]
 
+  CHAT_CHANNEL_PARAMS = %i[
+    per_page
+    page
+    channel_type
+    channel_status
+    status
+    user_id
+  ].freeze
+
```

Comment:
Since we're in here, may as well move these to a constant for consistency and performance 🚀

Reply:
Now since we're in here already, do we maybe also want to sort them alphabetically so your OCD coworker can sleep better at night? 😉